src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
```

Comment:
What is result? Let's use more descriptive names
You can also save `days * 40` instead of 0.